proposals/lip-0070.md

```diff
@@ -12,39 +12,39 @@ Required: 0022, 0023, 0057
 
 ## Abstract
 
-We define an on-chain reward sharing mechanism for the Lisk ecosystem. This mechanism is defined as an additional part of the [DPoS module][lip-0057]. In this LIP, we specify the state transitions logic defined within the DPoS module due to reward sharing, i.e. the commands, the protocol logic injected during the block lifecycle, and the functions that can be called from other modules or off-chain services.
+We define an on-chain reward sharing mechanism for the Lisk ecosystem. This mechanism is defined as an additional part of the [PoS module][lip-0057]. In this LIP, we specify the state transitions logic defined within the PoS module due to reward sharing, i.e. the commands, the protocol logic injected during the block lifecycle, and the functions that can be called from other modules or off-chain services.
 
 ## Copyright
 
 This LIP is licensed under the [Creative Commons Zero 1.0 Universal](https://creativecommons.org/publicdomain/zero/1.0/).
 
 ## Motivation
 
-In Lisk protocol, delegates receive rewards for generating blocks. The set of delegates that generate blocks is chosen based on the total voted amount (self-votes and votes from voters), according to the selection mechanism specified in [LIP 0022][lip-0022] and [LIP 0023][lip-0023]. In contrast to delegates, voters do not receive any rewards by the protocol for locking their tokens to the voted delegates; in other words, there are no economic incentives for users to participate in voting. 
+In Lisk protocol, validators receive rewards for generating blocks. The set of validators that generate blocks is chosen based on the total staked amount (self-stakes and stakes from stakers), according to the selection mechanism specified in [LIP 0022][lip-0022] and [LIP 0023][lip-0023]. In contrast to validators, stakers do not receive any rewards by the protocol for locking their tokens to the staked validators; in other words, there are no economic incentives for users to participate in staking. 
```

Comment:
staked validators?? Should we use "voted validators" here?

Reply:
cbr

Reply:
rephrased, have a look